app/views/recipes/_recipe_details.html.erb

```diff
@@ -0,0 +1,56 @@
+<div class="container shadow p-5 recipe_details">
+  <h1 class="text-center mb-4"><%= @recipe.name %></h1>
+
+  <div class="d-flex justify-content-between align-items-center mb-4">
+    <div>
+      <span class="h5">Preparation time:</span>
+      <span> <%= @recipe.preparation_time %></span>
+    </div>
+    <div>
+      <div class="form-check form-switch">
+        <input disabled class="form-check-input" type="checkbox" id="toggle-public" <%= @recipe.public ? "checked" : "" %>>
+        <label class="form-check-label" for="toggle-public">Public</label>
+      </div>
```

Comment:
- [ ] I recommend you to make this public button functional so user can change it after creating the recipe and would make more sense.